src/report/graphs.py

```diff
@@ -299,65 +390,93 @@ def plot(self, filename, size=1.0):
         if self.final_bucket_accumulate:
-            tick_labels[-1] = '{}+'.format(tick_labels[-1])
-
-        plt.bar(pos, self.series.values, tick_label=tick_labels,
-                align='center', color='#000000', edgecolor='#000000')
+            tick_labels[-1] = "{}+".format(tick_labels[-1])
+
+        plt.bar(
+            pos,
+            self.series.values,
+            tick_label=tick_labels,
+            align="center",
+            color="#000000",
+            edgecolor="#000000",
+        )
         y_max = ax.get_ylim()[1]
 
         # Colorize regions and add dividing lines if region_colors
         # present
         previous_day = 0
         for (day, bgcolor) in self.region_colors:
             # draw reference lines
-            plt.axvline(x=day, color='#777777', linewidth=0.5)
-            ax.annotate('{} Days '.format(day), xy=(day - 1, y_max),
-                        rotation='vertical', fontsize=7, color='#666666',
-                        ha='right', va='top')
+            plt.axvline(x=day, color="#777777", linewidth=0.5)
+            ax.annotate(
+                "{} Days ".format(day),
+                xy=(day - 1, y_max),
+                rotation="vertical",
+                fontsize=7,
+                color="#666666",
+                ha="right",
+                va="top",
+            )
             ax.add_patch(
-                Rectangle((previous_day, 0), day - previous_day, y_max,
-                          facecolor=bgcolor, alpha=0.4, edgecolor=None,
-                          zorder=0))
+                Rectangle(
+                    (previous_day, 0),
+                    day - previous_day,
+                    y_max,
+                    facecolor=bgcolor,
+                    alpha=0.4,
+                    edgecolor=None,
+                    zorder=0,
+                )
+            )
             previous_day = day
-        ax.add_patch(Rectangle((previous_day, 0),
-                               (self.series.index[-1] - previous_day +
-                                self.x_limit_extra),
-                               y_max,
-                               facecolor='#000000', alpha=0.4, edgecolor=None,
-                               zorder=0))
+        ax.add_patch(
+            Rectangle(
+                (previous_day, 0),
+                (self.series.index[-1] - previous_day + self.x_limit_extra),
+                y_max,
+                facecolor="#000000",
+                alpha=0.4,
+                edgecolor=None,
+                zorder=0,
+            )
+        )
 
         tick_interval = len(self.series) / (self.x_major_tick_count - 1)
         for i, tick in enumerate(ax.xaxis.get_major_ticks()):
             if i % tick_interval:
                 tick.set_visible(False)
             else:
                 tick.set_visible(True)
-                tick.set_label('{}'.format(self.series.index[i]))
+                tick.set_label("{}".format(self.series.index[i]))
```

Comment:
Is there a reason we're doing it this way? I'm not sure if a `str()` wrapper is even necessary, but this `format()` call seems unnecessary.

Reply:
See commit 6d76a32.